.github/workflows/docs.yml

```diff
@@ -1,21 +1,54 @@
+# Workflow for updating the documentation website
 ---
-name: Sync `docs` directory to ReadMe
+name: Update Documentation
+
+# Documentation updates can happen in different ways; allow only one at a time
+concurrency: release
 
 on:
+  # Allow running this workflow manually from the Actions tab
   workflow_dispatch:
   push:
     branches:
       - main
-  
+
 jobs:
   Update-Documentation:
     name: Update the documentation
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
```

Comment:
`bash` is already [the default][1]

[1]: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell

```suggestion
```